test/AppStatsTest.php

```diff
@@ -31,6 +31,8 @@ protected function setUp() {
     }
 
     public function testPublishEventsForwards() {
+        echo '== testPublishEventsForwards()';
```

Comment:
Ah, I was not aware, not nice but perhaps leave as is then if it's only in the tests.